### DISCUSSION TITLE
RE2 - Rooomscale and head aiming and using framework options

### DISCUSSION THREAD

**Discussion by alistairgd** [praydog/REFramework#591] [General Questions]
Trying hard to understand and tweak RE2 VR to comfortable gameplay - appreciate any help... 
Playing in Vive Pro with X-Box Controller

1. What does the Roomscale option do - I could not see any difference?
2. Playing in 1st person, aiming seems to be done with the head movement, is there another way, ie with the controller?
3. I'm selecting VR Framework options (eg 1st person) on the PC monitor, then putting the headset on, is there a way to select the framework options with the HMD on?
4. Is there a way to reset the height from in the game, as I'm always floating above the floor?

Thanks.

**Comment by praydog** [comment]
It causes your character to follow the headset movement. If you look down you'll see your legs slide along with you.

You have to actively be using your Vive controllers for the "gamepad mode" to not kick in. Otherwise there is no way to turn it off.

https://cursey.github.io/reframework-book/troubleshooting/VR-Troubleshooting.html

**Reply by alistairgd** [reply]
Okay, thanks, I will experiment further with wands / index controllers.

And thanks for your work on this, you have created a whole new longevity to some of our favourite games.
